src/promptflow/promptflow/_sdk/_submitter/test_submitter.py

```diff
@@ -205,6 +207,7 @@ def init(
         environment_variables: Optional[dict] = None,
         stream_log: bool = True,
         output_path: Optional[str] = None,
+        port: Optional[int] = None,
```

Comment:
Could proxy control executor service in another host?

Reply:
got your point. let me update

Reply:
+ @0mza987 Honglin, your chat group feature might also need this